frontend/src/pages/About/About.jsx

```diff
@@ -40,7 +40,7 @@ export const About = (props) => {
       >
         <div className="row">
           <div className="col-lg-5">
-            <h1>Who are we and what we do?</h1>
+            <h1 className="first">Who are we and what we do?</h1>
```

Comment:
please make it stick to the left similar to [this](https://github.com/HITK-TECH-Community/Community-Website/pull/744/files#diff-9629027d328ffb062e91696c8962c46a51ad143e1f6224b2473a6afd51c540f3R52) line and same goes for the heading [resent in the home page